### DISCUSSION TITLE
Map table back to object

### DISCUSSION THREAD

**Discussion by maslewandowski** [ClosedXML/ClosedXML#2255] [Q&A]
I would like to know, if there is an easy way to map a datatable back to it's object.

I create a datatable for the worksheet with this code:
`var dataTable = ws.Cell("A1").InsertTable(positions, "Positionen");`

Is there a similar way to get the object back? something like
`var positions = ws.Table("Positionen").MapToObject();`

Thanks for help

**Comment by igitur** [comment]
There is a `IXLTable.AsNativeDataTable()` method that would get your partly there. Converting that to strongly typed records should be trivial. I'm guessing there's a micro ORM that can help with that.
